sdk/containerregistry/azure-containerregistry/CHANGELOG.md

```diff
@@ -1,15 +1,21 @@
 # Release History
 
-## 1.0.0b8 (Unreleased)
+## 1.0.0 (2022-01-19)
```

Comment:
You're planning to release today?
The APIview has been approved for GA.

Reply:
Thanks for bringing this up! This was the plan, now I think there just has some quick updates, planning to release tomorrow(2022-01-20).